packages/plugins/videos-ui/src/lib/shared/ui/soundshot/sounshot-player-skeleton/sounshot-player-skeleton.component.html

```
@@ -0,0 +1,25 @@
+<div class="skeleton-container" [class.animate]="animate">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)